Libplanet/Blockchain/BlockChain.cs

```diff
@@ -174,13 +174,16 @@ IEnumerable<IRenderer<T>> renderers
 
             if (Count == 0)
             {
-                Append(
-                    genesisBlock,
-                    currentTime: genesisBlock.Timestamp,
-                    renderBlocks: !inFork,
-                    renderActions: !inFork,
-                    evaluateActions: !inFork
-                );
+                if (!inFork)
```

Comment:
How merge if statements?
```suggestion
                if (count == 0 && !inFork)
```

Reply:
It will throw exception in next `elseif` statement 😢